05-classification.Rmd

```diff
@@ -963,12 +988,14 @@ for the number of neighbours and weight function (for now, these are just the va
  manually when we created `knn_spec` above). But now the fit object also includes information about
 the overall workflow, including the centering and scaling preprocessing steps.
 
-Let's visualize the predictions that this trained K-nearest neighbour model will make on new observations.
-Below you will see how to make the coloured prediction map plots from earlier in this chapter.
+Figure \@ref(fig:05-workflow-plot-show) visualizes the predictions that this 
+trained K-nearest neighbour model will make on new observations.
+Although you have seen coloured prediction map visualizations like this a few times now,
+we have not included the code to generate them, as it is a little bit complicated.
+For the interested reader who wants a learning challenge, we now include it below. 
 The basic idea is to create a grid of synthetic new observations using the `expand.grid` function, 
 predict the label of each, and visualize the predictions with a coloured scatter having a very high transparency 
-(low `alpha` value) and large point radius. We include the code here as a learning challenge; see
-if you can figure out what each line is doing!
+(low `alpha` value) and large point radius. See if you can figure out what each line is doing!
 
 ```{r 05-workflow-fit-plot} 
 # create the grid of area/smoothness vals, and arrange in a data frame
```

Comment:
A few of the lines of code below are too long to render in the LaTeX format.

Reply:
created a new global pass issue for this #200